airflow/providers/cncf/kubernetes/hooks/kubernetes.py

```diff
@@ -845,6 +845,7 @@ async def wait_until_container_complete(
         :param name: Name of Pod to fetch.
         :param namespace: Namespace of the Pod.
         :param container_name: name of the container within the pod to monitor
+        :param poll_interval: Interval in seconds between polling the job status
```

Comment:
please extract these docstring changes to separated PR. they are not relevant to the google fix

Reply:
Sure that makes sense , let me updated :)

Reply:
updated :)